CHANGES.rst

```diff
@@ -18,6 +18,7 @@ Improvements
   to avoid collisions and inconsistencies (:pr:`5478`)
 - Add option to use review track as accepted track when bulk-accepting abstracts
   (:pr:`5608`)
+- Add CC field on contribution email dialogs (:pr:`5637`)
```

Comment:
I just noticed that we use BCC for those addresses, so I think we should use BCC consistently her and in the UI as well. (I assumed it was CC when we talked about this before)